### PR TITLE
Fixed mapping of double amounts

### PR DESCRIPTION
**Pull request by KDNeufeld**
Fixed mapping of Doubles from ORDS.

### CODE REVIEW

File: src/backend/oracle-data-api/src/main/resources/tco-openapi-spec.yaml [hunk 1]
```diff
@@ -539,9 +539,9 @@ components:
         suspSntcProbationDurtnTxt:
           type: string
         ticketedAmt:
-          type: double
+          type: string
```
Comment (pbolduc) [inline src/backend/oracle-data-api/src/main/resources/tco-openapi-spec.yaml:542]: Are these text fields in Oracle?

**Review by pbolduc — APPROVED** [review]
Only odd thing is using strings for the numeric amounts.

File: src/backend/oracle-data-api/src/main/resources/tco-openapi-spec.yaml (hunk 1 above)
Comment (KDNeufeld) [reply src/backend/oracle-data-api/src/main/resources/tco-openapi-spec.yaml:542]: This change is when the string is getting converted to numeric.  The datatype is still numeric in the application, but we were getting errors with Jackson specifically if mapped here.  After Jackson, Mapstruct converts this ticketedAmt column to numeric.

**Comment by KDNeufeld** [comment]
> Only odd thing is using strings for the numeric amounts.

Agreed. The application is still using numeric, we're just changing where the conversion from string to numeric occurs -> from Jackson to Mapstruct.

**Pull request merged**